index.html

```diff
@@ -10,219 +10,268 @@
 </head>
 
 <body>
-    <div class="main-wrapper">
-        <header class="toolbar">
-            <div class="logo-place">
-                <a href="./index.html" class="my-logo hex">My Logo</a>
+    <section class="main-wrapper">
+            <header class="toolbar">
+                <div class="logo-place">
+                    <a href="./index.html" class="my-logo hex">My Logo</a>
+                </div>
+                <nav>
+                    <figure class="icons">
+                        <img src="images/Icons/Union.png" alt="Icons" />
+                    </figure>
+    
+                    <ul class="nav-bar-ul">
+                        <a href="#"><li>Portoflio</li></a>
+                        <a href="#"><li>About</li></a>
+                        <a href="#"><li>Contact</li></a>
+                      </ul>
+                </nav>
+            </header>
+            <div class="headline">
+                <h1>I’m Jonathan<br> Glad to see you!</h1>
+                
+                <p class="text">
+                    I’m a software developer! I can help you build a product , feature or
+                    website Look through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact
+                    me.
+                </p>
+                <h2 class="hex connect">LET’S CONNECT</h2>
+                <ul class="social-media">
+                    <li class="socials">
+                        <a href="#"><img src="images/socials/twitter.svg" /></a>
+                    </li>
+                    <li clalss="socials">
+                        <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+                    </li>
+                    <li clalss="socials">
+                        <a href="#"><img src="images/socials/m.svg" /></a>
+                    </li>
+                    <li clalss="socials">
```

Comment:
<img width="983" alt="image" src="https://user-images.githubusercontent.com/41163571/230602695-40af5e2a-c106-4496-a558-18a88ccc45db.png">

- [ ] Kindly ensure that the container spans the appropriate with like it is in the Figma design to have a pixel perfect design 😉.